ddp-workspace/projects/ddp-pancan/src/app/components/faq-section/faq-section.component.scss

```diff
@@ -0,0 +1,52 @@
+$border-color: #EBEBEB;
+$icon-width: 24px;
+$icon-margin-right: 2rem;
+$question-padding: 1.5rem;
+
+.faq-block__icon {
+    font-size: 30px;
+    margin-right: $icon-margin-right;
+}
+
+.faq-block__question {
+    margin: 0;
+}
+
+.faq-block__text {
+    white-space: pre-wrap;
+}
+
+.mat-expansion-panel {
+    border-top: 2px solid $border-color;
+    font-family: Untitled-Sans-Regular;
+    font-size: 24px;
+
+    &:last-child {
+        border-bottom: 2px solid $border-color;
+    }
+
+    :hover, :focus {
+        background-color: rgba(0,0,0,.04);
+    }
+
+    .mat-icon,
+    .faq-block__question,
+    .faq-block__text {
+        background-color: transparent;
```

Comment:
in order to not add this styles instead
:hover, :focus {...}
you need to set
&:hover, &:focus {...}
then the hover focus styles will be applied only to the parent .mat-expansion-panel but not to all elements. Anyway I hope all that styles will be removed once we fix material default styles

Reply:
see https://github.com/broadinstitute/ddp-angular/pull/631#discussion_r653475577